src/api-serverless/src/proxies/proxy.api.service.ts

```diff
@@ -0,0 +1,187 @@
+import { Profile } from '../../../entities/IProfile';
+import { BadRequestException } from '../../../exceptions';
+import { Logger } from '../../../logging';
+import {
+  profilesService,
+  ProfilesService
+} from '../../../profiles/profiles.service';
+import { Time } from '../../../time';
+import { CreateNewProfileProxy } from '../generated/models/CreateNewProfileProxy';
+import { ProfileProxyEntity } from '../../../entities/IProfileProxy';
+import { randomUUID } from 'crypto';
+import {
+  ProfileProxiesDb,
+  profileProxiesDb
+} from '../../../profile-proxies/profile-proxies.db';
+import { ConnectionWrapper } from '../../../sql-executor';
+import { ProfileAndConsolidations } from '../../../profiles/profile.types';
+import { Page } from '../page-request';
+
+export class ProfileProxyApiService {
```

Comment:
IMO you should merge ProfileProxyApiService and ProfileProxyActionApiService into one service. Same applies to those DB's.